src/Evolution/Initialization/AddToDataBox.hpp

```diff
@@ -31,23 +31,24 @@ namespace detail {
 template <typename AddingAction, typename SimpleTag, MergePolicy Policy,
           typename DbTagsList,
           Requires<Policy == MergePolicy::Overwrite> = nullptr>
-void check_simple_tag_value(const gsl::not_null<db::DataBox<DbTagsList>*> box,
-                            db::item_type<SimpleTag>&& simple_tag) noexcept {
+void merge_simple_tag_value(
+    const gsl::not_null<db::DataBox<DbTagsList>*> box,
+    db::item_type<SimpleTag>&& simple_tag_value) noexcept {
   db::mutate<SimpleTag>(
       box,
-      [](const auto item_ptr,
-         db::item_type<SimpleTag>&& local_simple_tag) noexcept {
-        *item_ptr = std::move(local_simple_tag);
+      [](const gsl::not_null<db::item_type<SimpleTag>*> stored_simple_tag_value,
+         db::item_type<SimpleTag>&& local_simple_tag_value) noexcept {
+        *stored_simple_tag_value = std::move(local_simple_tag_value);
       },
-      std::move(simple_tag));
+      std::move(simple_tag_value));
 }
 
 template <
     typename AddingAction, typename SimpleTag, MergePolicy Policy,
     typename DbTagsList,
     Requires<(Policy == MergePolicy::Error or Policy == MergePolicy::Ignore) and
```

Comment:
What do you mean with "Ignore":
- Skip any SimpleTags that are already in the DataBox, not overwriting them.
- Skip only those SimpleTags that can't be compared for equivalence.

Currently you implement the second option, but I would expect the first since that's the opposite of what "Overwrite" does.

Reply:
Please see the documentation. This does exactly what it is documented to do.

Reply:
It does. What I meant is, I would expect that specifying "MergePolicy::Ignore" would actually ignore simple tags that have conflicts. Instead, it only ignores those that can't be compared. The current implementation doesn't allow for the case where you don't want the initialiser to overwrite the existing value.

Reply:
I don't think it's ever the correct answer to completely ignore the existing tags. That seems like it's asking for a bug so I'd rather not add an option that, at least to me currently, seems only useful for introducing bugs into the code. If you have a recommended alternative name I'm happy to rename the enum option.

Reply:
Perhaps make it `IgnoreIncomparable` or something (I know, it's about equivalence not comparability, but can't think of a better name). With the current implementation it's probably a pretty extreme edge-case, so the name won't be used a lot and can be long and descriptive. Most importantly, please fix the documentation for the enum case, which currently suggests this would actually ignore existing simple tags.

You could also just remove that case until we need it (if ever), and just error out if you can't compare the values.